tests/fault_injector/conftest.py

```diff
@@ -49,9 +49,16 @@ def event_bus(run):
     return bus
 
 
+class MockRouteController:
+    method_calls: int = 0
+
+    def recalculate_all_routes(self):
+        self.method_calls += 1
+
+
 @pytest.fixture
 def interlocking():
```

Comment:
Here again, I do not realy like the naming. But it is not that important.

Reply:
I changed it to `interlocking_disruptor`